### PR TITLE
Preemptible flag for GCE

### PR DESCRIPTION
**Pull request by kron4eg**
```release-note
Preemptible flag for GCE
```

### CODE REVIEW

File: pkg/cloudprovider/provider/gce/config.go [hunk 1]
```diff
@@ -170,6 +172,11 @@ func newConfig(resolver *providerconfig.ConfigVarResolver, spec v1alpha1.Provide
 		return nil, fmt.Errorf("cannot retrieve subnetwork: %v", err)
 	}
 
+	cfg.preemptible, err = resolver.GetConfigVarBoolValue(cpSpec.Preemptible)
+	if err != nil {
+		return nil, fmt.Errorf("cannot retrieve preemptive: %v", err)
```
Comment (alvaroaleman) [inline pkg/cloudprovider/provider/gce/config.go]: typo

Comment (kron4eg) [reply pkg/cloudprovider/provider/gce/config.go]: ✔️

**Comment by kron4eg** [comment]
> Please add the `preemtible` option to the example in `examples/gce-machinedeployment.yaml`
done

**Comment by alvaroaleman** [comment]
/lgtm
/approve 

You can btw use `/retest` to re-run all failed tests, no reason to explicitly call the test by name

**Pull request merged**